Changes for RHDEVDOCS-4829 Doc: ODC Usability & Customer Happiness Epic (#55909)
openshift/openshift-docs: modules/odc-creating-helm-releases-using-developer-perspective.adoc

Where available, you can switch between the *YAML View* and *Form View*. The data is persisted when switching between the views.
====
+

> .. Click *Create* to create a Helm release. You will be redirected to the *Topology* view where the release is displayed. If the Helm chart has release notes, the chart is pre-selected and the right panel displays the release notes for that release. If a Helm chart creates workloads when installed, then the user is redirected to *Topology*, or else is navigated to the *Helm release details* page. The workloads are DaemonSet, CronJob, Pod, Deployment, and DeploymentConfig.

Reviewer: ```suggestion
.. Click *Create* to create a Helm release. The <console?> displays the new release in the *Topology* view. 
+
If a Helm chart has release notes, the <console?> displays them. 
+
If a Helm chart creates workloads, the <console?> displays them on the *Topology* or *Helm release details* page. The workloads are `DaemonSet`, `CronJob`, `Pod`, `Deployment`, and `DeploymentConfig`.
```
- For clarity, avoid switching between different forms of address. For example, the content switches between addressing the user as "you" and "the user". Rather, use the dominant form of address consistently. 
- Use present tense; avoid future tense ("will").
- Evaluate removing/minimizing descriptions of the outcome of the step. For example, here, consider moving "The workloads are `DaemonSet`, `CronJob`, `Pod`, `Deployment`, and `DeploymentConfig`." to the topic overview or related a concept topic. The user might get this information from the screen when it happens. (However, if this information is essential for the user to see at that moment, for example, if it affects the decisions and actions, you can keep it there.)

Author: Updated this. I have used "web console" in place of the console placeholders you have mentioned. Thank you for help.
